moderngl_window/atlas/base.py

```diff
@@ -44,21 +45,21 @@ class AtlasImage(BaseImage):
     """An atlas image using Pillow"""
 
     def __init__(self, image: Image):
-        self.image = image
+        self._image = image
```

Comment:
I think the underscore was forgotten here

Reply:
Yup. It's sadly incomplete code I need to clean up soon